app/src/main/AndroidManifest.xml

```diff
@@ -1,15 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.feedhenry.helloworld_android"
-    android:versionCode="1"
-    android:versionName="1.0" >
+    package="com.feedhenry.helloworld_android">
 
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
 
-    <uses-sdk
-        android:minSdkVersion="8"
-        android:targetSdkVersion="19" />
```

Comment:
Gradle don't care for `AndroidManifest.xml` all version configs are in [build.gradle](https://github.com/feedhenry-templates/helloworld-android-gradle/blob/master/app/build.gradle#L7-L13)